assets/js/components/Sidebar.vue

```diff
@@ -1,117 +1,35 @@
 <template>
-  <div id="sidebar-results" class="sidebar sidebar-results" v-if="hasData">
-    <div class="sidebar-header">
-      <transition name="fade">
-        <div class="allocation" :key="transitioned">
-          <strong>{{ format(data.allocation) }}</strong>
-          <small>{{ format(data.bilateral_allocation) }} for bilateral relations</small>
-        </div>
-      </transition>
-      <button type="button" id="close-sidebar-results" class="no-btn"
-              title="Close results"
-              v-if="isMobileExpanded"
-              v-on:click="mobileCollapse">
-        <span class="icon icon-close"></span>
-      </button>
-    </div>
-
-    <div class="sidebar-tabs">
-      <nav class="sidebar-tab-menu">
-        <a class="sidebar-tab-menu-item"
-           v-on:click="selectTab('results')"
-           v-bind:class="{ active: (selectedTab == 'results') }">
-         <small>Results</small>
-        </a>
-        <a class="sidebar-tab-menu-item"
-           v-on:click="selectTab('programmes')"
-           v-bind:class="{ active: (selectedTab == 'programmes') }">
-          <small>Programmes</small>
-        </a>
-      </nav>
-    </div>
-
-    <div class="sidebar-tab-content">
-      <results id="results"
-               :initial="dataset"
-               v-bind:selected="(selectedTab == 'results')"/>
-      <programmes id="programmes"
-                  :initial="dataset"
-                  v-bind:selected="(selectedTab == 'programmes')"/>
-    </div>
+  <div class="sidebar">
+    <button type="button" id="close-sidebar" class="no-btn"
+                  title="Close results"
+                v-if="isMobileExpanded"
+                v-on:click="mobileCollapse">
+          <span class="icon icon-close"></span>
+  </button>
+    <slot name="Allocation_overview"></slot>
+    <slot name="Tabs"></slot>
```

Comment:
nope, wrong. why the two slots? if you make the sidebar aware of what's inside it, then you'll end up needing one sidebar component for each scenario. wrong.